### PR TITLE
[DOCS] updated docs structure

### PR DESCRIPTION
**Pull request by sidohaakma**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] User documentation updated
- [x] (If you have changed REST API interface) view-swagger.ftl updated
- [x] Test plan template updated
- [x] Clean commits
- [x] Added Feature/Fix to release notes

### CODE REVIEW

File: docs/background.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# What is MOLGENIS?
```
Comment (fdlk) [inline docs/background.md]: Duplicates README.md, is this necessary?

File: docs/guide-explore.md [hunk 2]
```diff
@@ -58,20 +63,50 @@ Same as the add row button, but with prefilled inputs to allow you to edit a row
...
-This button can be used to remove a row from the entity. 
+This button can be used to remove a row from the entity.
+
+### Report view
 
-### System packages, system entities and system attributes
+Optionally, custom reports can be available for the selected table. 
 
-System packages, system entities and system attributes are available in MOLGENIS 2.0 after installation. They are configured programmatically and can only be changed in new releases of the software. Changing the system packages, system entities and system attributes via the metadata editing functionality, will be registerd in the database and update MOLGENIS (not the java part). When dropping the index and database, this registration will be dropped, and the original values will be loaded again.
+The reports functionality is made for overriding the default instance view or to add an instances tab in the Data-explorer. Overriding the views or adding a tab is possible by creating a new `FreemarkerTemplate` entity with the right name convention. This short tutorial will show you how to achieve this.
```
Comment (fdlk) [inline docs/guide-explore.md]: The reports and aggregates got mixed up here.

File: docs/guide-explore.md [hunk 3]
```diff
@@ -172,52 +184,37 @@ You now get a 1 dimensional list of counts, showing you that every patient has 3
...
 ###### Override the entity view
 
 Steps:
 
 1. Go to the Data Explorer.
 2. Select the "cities" entity via the entity select dropdown.
-3. The entity view modal is opened when you click on the ![View entity report button](../../images/reports/view-entityreport-button.png?raw=true, "Entity view") button".
-4. The default view will be: ![View entity report default](../../images/reports/default-entityreport-view.png?raw=true, "Entity view")
-5. Let's upload our own template. 
+3. The entity view modal is opened when you click on the ![View entity report button](images/reports/view-entityreport-button.png?raw=true, "Entity view") button".
```
Comment (fdlk) [inline docs/guide-explore.md]: And here's the second half of the reports

File: docs/guide-explore.md [hunk 4]
```diff
@@ -58,20 +63,50 @@ Same as the add row button, but with prefilled inputs to allow you to edit a row
...
+Optionally, custom reports can be available for the selected table. 
 
-System packages, system entities and system attributes are available in MOLGENIS 2.0 after installation. They are configured programmatically and can only be changed in new releases of the software. Changing the system packages, system entities and system attributes via the metadata editing functionality, will be registerd in the database and update MOLGENIS (not the java part). When dropping the index and database, this registration will be dropped, and the original values will be loaded again.
+The reports functionality is made for overriding the default instance view or to add an instances tab in the Data-explorer. Overriding the views or adding a tab is possible by creating a new `FreemarkerTemplate` entity with the right name convention. This short tutorial will show you how to achieve this.
 
-# Download  
+There are two ways to create your own reports:
 
-At the bottom right of each table, there is a download button. This button allows you to save the data to a CSV or XLSX file. The XLSX download is limited to 500000 values to prevent performance issues. 
+1. Overriding the default instance view.
```
Comment (fdlk) [inline docs/guide-explore.md]: This is aggregates again.

File: docs/guide-metadata-manager.md [hunk 5]
```diff
@@ -1,26 +1,26 @@
-# Metadata manager
-The metadata manager can be used to manage your metadata.
+# Data model editor
```
Comment (fdlk) [inline docs/guide-metadata-manager.md]: Wow! yet another name :D

File: docs/SUMMARY.md [hunk 6]
```diff
@@ -1,74 +1,71 @@
...
-    * [Creating bootstrap themes](developer_documentation/creating-themes.md)
-  * Backend  
-    * [Security](developer_documentation/security.md)
-    * [Data Backend](developer_documentation/backend.md)
-    * [Services](developer_documentation/service.md)
-    * [Jobs](developer_documentation/jobs.md)
-    * [Dynamic Decorators](developer_documentation/dynamic-decorators.md)
-  * Testing
-    * [Running the integration tests](developer_documentation/integration-tests.md)
+  * [What is MOLGENIS](background.md)
```
Comment (fdlk) [inline docs/SUMMARY.md]: These strings do not match the page headers

File: docs/guide-authentication.md [hunk 7]
```diff
@@ -27,65 +28,65 @@ You have to scan the QR-code with an authenticator-app. Examples of authenticato
...
 
-![two-factor authentication recoverycodes](../../../images/molgenis_two-factor-authentication_recoverycodes.png?raw=true, "two-factor authentication recoverycodes")
+When the QR-code is scanned, your authenticator-app will create an account for MOLGENIS and also generate a verification code for that account.
+You have to fill in the verification code in de box below the QR-code. If you have entered the verification code you will be redirected to the
+*Account-Security*-tab. This will show the recovery-codes.
+
+![two-factor authentication recoverycodes](images/molgenis_two-factor-authentication_recoverycodes.png?raw=true, "two-factor authentication recoverycodes")
 
-Make sure you store the recovery codes somewhere (not in MOLGENIS) so you can access them when you for example loose your phone (see [TROUBLESHOOTING](#TROUBLESHOOTING)). 
+Make sure you store the recovery codes somewhere (not in MOLGENIS) so you can access them when you for example lose your phone (see [TROUBLESHOOTING](#TROUBLESHOOTING)).
```
[the remark this answers is not in the document]
Comment (fdlk) [reply docs/guide-authentication.md]: Found out that they do work cross-page so I suspect you'll need to add the page to the link

File: docs/background.md (hunk 1 above)
Comment (sidohaakma) [reply docs/background.md]: For gitbook it is. We could leave the background.md out of this PR for now?

Comment (sidohaakma) [reply docs/background.md]: It is used in vuepress.

Comment (sidohaakma) [reply docs/background.md]: I deleted it.

File: docs/guide-api-rest2.md [hunk 8]
```diff
@@ -26,14 +24,7 @@ POST http://molgenis.mydomain.example/api/v2/<entity_name>/<entity_id>?_method=G
 attrs=attr0(subattr0,subattr1),attr1(*)
 ```
 
-## Delete instance
```
[the remark this answers is not in the document]
Comment (sidohaakma) [reply docs/guide-api-rest2.md]: Placed it back.

File: docs/guide-app-development.md [hunk 9]
```diff
@@ -1,9 +1,6 @@
-# Creating apps for MOLGENIS
-To be able to host your app inside MOLGENIS, you need to create an archive in a specific format.
+# Apps
 
-To get started right away, you can download an [example archive here](../data/example-app.zip).
-
-Following paragraphs will explain in detail how to construct and configure your resources to make it a certified MOLGENIS app.
+You can create add-on user interfaces using html+javascript. To be able to host your app inside MOLGENIS, you need to create an archive in a specific format. To get started right away, you can download an [example archive here](../data/example-app.zip). Following paragraphs will explain in detail how to construct and configure your resources to make it a certified MOLGENIS app.
```
[the remark this answers is not in the document]
Comment (sidohaakma) [reply docs/guide-app-development.md]: is fixed.

File: docs/guide-authentication.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (sidohaakma) [reply docs/guide-authentication.md]: Removed link.

**Pull request merged**